internal/cmd/skupper/token/kube/token_issue.go

```diff
@@ -112,7 +77,7 @@ func (cmd *CmdTokenIssue) ValidateInput(args []string) []error {
 	if siteList == nil || len(siteList.Items) == 0 {
 		validationErrors = append(validationErrors, fmt.Errorf("A site must exist in namespace %s before a token can be created", cmd.namespace))
 	} else {
-		if !utils.SiteConfigured(siteList) {
+		if !utils2.SiteConfigured(siteList) {
```

Comment:
Can be changed later:    But we should be checking site is active instead of configured here

Reply:
Issue created: #1614